docs/images/zip-0317.tex

```diff
@@ -0,0 +1,20 @@
+\documentclass[]{article}
```

Comment:
Why don't we just link to the ZIP 317 page?  I don't think we need to check in a 120 KB png file either.

Reply:
Ooooh. I was reading at https://github.com/zcash/zips/blob/main/zip-0317.rst, where the tex doesn't render